services/order.py

```diff
@@ -0,0 +1,37 @@
+from db.models import Order, Ticket
+from django.db import transaction
+from django.contrib.auth import get_user_model
+from typing import List, Dict
+
+User = get_user_model()
+
+
+@transaction.atomic
+def create_order(
+        tickets: List[Dict],
+        username: str,
+        date: str = None
+) -> Order:
+    user = User.objects.get(username=username)
+    order = Order.objects.create(user=user)
+
+    if date:
+        order.created_at = date

```

Comment:
Consider using a `datetime` object for the `date` parameter instead of a `str` for better type safety and to leverage Django's date/time handling capabilities.